README.md

```diff
@@ -100,13 +125,15 @@ Password (again):
 Superuser created successfully.
 ```
 
-Using your admin credentials, verify that you are able to log into the admin django site, which should be located in http://localhost:8000/admin/
+Using your admin credentials, verify that you are able to log into the admin django site, which should be located at `/admin/`. 
 
 ![image](https://user-images.githubusercontent.com/7258463/101060564-8be1e500-355d-11eb-887f-5af65b50ba13.png)
 
+When initializing the site, remember to apply migrations to the database (see )
```

Comment:
`(see )`?

Reply:
ugh...meant to fix that. Thanks for catching!